docs/README-macOS.md

```diff
@@ -0,0 +1,15 @@
+This README provides a step-by-step guide on how to run ramalama on macOS.
+
+```bash
+brew install go
+brew install podman
```

Comment:
It can be much simpler than this, install podman cli or podman desktop for macOS here:

https://podman.io/

this official installer comes with krunkit for AI acceleration, etc. I'd avoid recommending brew podman, it's not as good.

Change the hypervison to krunkit. Run podman machine init, podman machine start, etc.

Then run:

curl -fsSL https://raw.githubusercontent.com/containers/ramalama/s/install.sh | bash

The "curl" command is the only one you strictly need if you don't want to use containers.

Reply:
Agreed.  `brew install podman` is not recomended. Link to installation of podman desktop and recommend libkrun

Reply:
Also podman should be optional, Docker is an option.

Reply:
Sometimes I wish RamaLama was just part of podman desktop and podman cli installers. It's like 10 python files? Super tiny...